### ISSUE TITLE
During training, dump CSV of loss (instead of just the plot)

### ISSUE THREAD

**Issue by anandthakker** [developmentseed/skynet-train#14]
@drewbo 's one-liner:
```sh
cat train/train_0-10000.log | grep -E "Iteration [0-9]+, loss = [0-9\.]+" | sed -E "s/.* Iteration //g" | sed -E "s/ loss = //g" > training-loss.csv
```
